lib/src/kbasesearchengine/main/DecorateWithNarrativeInfo.java

```diff
@@ -0,0 +1,121 @@
+package kbasesearchengine.main;
+
+import kbasesearchengine.*;
```

Comment:
Please don't use wildcard imports

Reply:
Done. (It was there in the original code.)

Reply:
But DecorateWithNarrativeInfo was a new class...?